services/lab-service/traffexam/kilda/traffexam/context.py

```diff
@@ -71,7 +72,11 @@ def make_veth_base_name(self):
     def set_root(self, root):
         self.root = pathlib.Path(root)
         if not self._init_done:
-            self.root.mkdir(parents=True, exist_ok=True)
+            try:
+                os.makedirs(str(self.root))
```

Comment:
AFAIR `os.makedirs` has `exist_ok` flag as well why not to use it here?

Reply:
hm... I have missed the moment when it was added there.